virtool/redis.py

```diff
@@ -74,3 +74,27 @@ async def check_version(redis: Redis) -> Optional[str]:
             return version
 
     return None
+
+
+async def resubscribe(redis: Redis, redis_channel_string: str) -> Channel:
```

Comment:
Don't put types in variable names: ``redis_channel_string`` to ``redis_channel_name``.

https://docs.quantifiedcode.com/python-anti-patterns/readability/putting_type_information_in_a_variable_name.html